src/utils.ts

```diff
@@ -72,6 +72,10 @@ export function cfGetConfigFilePath(): string {
     return path.join(_.get(process, "env.CF_HOME", os.homedir()), ".cf", "config.json");
 }
 
+export function isUpsType(resource: any): boolean {
```

Comment:
what is ups type ?

Reply:
it should be equal eServiceTypes.user_provided. (see line# 76 below)